### ISSUE TITLE
examples/scripts/sft.py fails

### ISSUE THREAD

**Issue by windmaple** [huggingface/trl#1349]
I was trying to finetune Gemma using the code from HF blog and it just failed. Also tried the code from sft.py comments and it failed w/ the same error. Here is the reproducing [notebook](https://colab.research.google.com/drive/1U7C8FRxacApxdHcSgTCsDQHnQw6ohRuj#scrollTo=171M16tZ8veD).

```
Traceback (most recent call last):
  File "/content/trl/examples/scripts/sft.py", line 112, in <module>
    peft_config=get_peft_config(model_config),
  File "/usr/local/lib/python3.10/dist-packages/trl/trainer/utils.py", line 718, in get_peft_config
    peft_config = LoraConfig(
NameError: name 'LoraConfig' is not defined. Did you mean: 'ModelConfig'?
```

@@xianbaoqian for visibility

**Comment by younesbelkada** [comment]
Hi @windmaple 
Thanks for the issue! 
Can you try to upgrade trl? `pip install -U trl`

**Comment by younesbelkada** [comment]
@windmaple I think I know where the issue comes from, in the notebook you need to install `peft`.
`pip install -U peft trl` should resolve it

**Comment by windmaple** [comment]
@younesbelkada thanks for the quick reply. You nailed it!

Could we add '!pip install bitsandbytes peft' to the finetune section of the [Gemma blog](https://huggingface.co/blog/gemma)? I'm sure it will help newbie like me when getting started.

**Comment by younesbelkada** [comment]
Thanks a lot @windmaple ! 🤩 
I merged #1350 to make the interface more userfriendly - OK will update it give me a moment

**Comment by xianbaoqian** [comment]
Haha one minute late.  Closing mine.
